app/config/AppConfig.scala

```diff
@@ -186,6 +189,7 @@ class FrontendAppConfig @Inject()(servicesConfig: ServicesConfig) extends AppCon
   lazy val nrsEnabled: Boolean = servicesConfig.getBoolean("feature-switch.nrsEnabled")
   lazy val crystallisationEnabled: Boolean = servicesConfig.getBoolean("feature-switch.crystallisationEnabled")
   lazy val tailoringEnabled: Boolean = servicesConfig.getBoolean("feature-switch.tailoringEnabled")
+  lazy val newTailoringEnabled: Boolean = servicesConfig.getBoolean("feature-switch.newTailoringEnabled")
```

Comment:
instead of newTailoring may be 
```suggestion
  lazy val phsae2TailoringEnabled: Boolean = servicesConfig.getBoolean("feature-switch.newTailoringEnabled")
```

Reply:
Decided to use tailorReturn* referring to repo, also put all the defs at one single place